flexmeasures/ui/templates/crud/account.html

```diff
@@ -1,183 +1,345 @@
-                            <th class="d-none">URL</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {% for asset in assets %}
-                        <tr>
-                            <td>
-                                <i class="{{ asset.generic_asset_type.name | asset_icon }} left-icon">{{ asset.name }}</i>
-                            </td>
-                            <td>
-                                {% if asset.latitude and asset.longitude %}
-                                LAT: {{ "{:,.4f}".format( asset.latitude ) }} LONG:
-                                {{ "{:,.4f}".format( asset.longitude ) }}
-                                {% endif %}
-                            </td>
-                            <td>
-                                {{ asset.id }}
-                            </td>
-                            <td>
-                                {% if asset.owner %}
-                                {{ asset.owner.name }}
-                                {% else %}
-                                PUBLIC
-                                {% endif %}
-                            </td>
-                            <td>
-                                {{ asset.sensors | length }}
-                            </td>
-                            <td>
-                                <a href="/assets/{{ asset.id }}/status">
-                                    <button type="button" class="btn">Status</button>
-                                </a>
-                            </td>
-                            <td class="text-right">
-                            </td>
-                            <td class="d-none">
-                                /assets/{{ asset.id }}
-                            </td>
-                        </tr>
-                        {% endfor %}
-                    </tbody>
-                </table>
-            </div>
-            </div>
+                <!-- Name color field -->
+                <div class="form-group">
+                  <div class="col-md-3">
+                    <label for="name" class="control-label">Name</label>
+                    <input
+                      type="text"
+                      class="form-control"
+                      id="name"
+                      name="name"
+                      value="{{ account.name }}"
+                      required
+                    />
+                  </div>
+                </div>
+
+                <!-- Primary color field -->
+                <div class="form-group">
+                  <div class="col-md-3">
+                    <label for="primary_color" class="control-label"
+                      >Primary Color</label
+                    >
+                    <input
+                      type="text"
+                      class="form-control"
+                      id="primary_color"
+                      name="primary_color"
```

Comment:
Let's add `title="The primary color to use when this platform is white-labelled and should show this account's branding."`